### ISSUE TITLE
[Sass Mixins] Hamburger mixin flooring breaks for rem units

### ISSUE THREAD

**Issue by tbredin** [zurb/foundation-sites#9099]
**How can we reproduce this bug?**

1. Call hamburger mixin using `rem` units, like so:
```scss
@include hamburger(
            $color: $primary-color,
            $color-hover: $primary-color,
            $width: 1.375rem,
            $height: 1.125rem,
            $weight: 0.2rem
        );
```
2. Spacing is floored to 0 in most cases (usually the spacing claculates as smaller than 1rem)

**What did you expect to happen?**
Didn't expect flooring to get applied when using non-pixel units.

**What happened instead?**
Spacing floored to 0rem.

**Comment by tbredin** [comment]
Proposed fix here: 
https://github.com/zurb/foundation-sites/compare/develop...tbredin:patch-1

**Comment by kball** [comment]
Fixed in #9100, will be in 6.2.4, scheduled to be released Oct 20.
